docs/user/openstack/README.md

```diff
@@ -33,7 +37,42 @@ clouds:
       auth_url: 'https://10.10.14.22:5001/v2.0'
 ```
 
-* Swift must be enabled.  The user must have `swiftoperator` permissions and
+If you choose to use environment variables in place of a clouds.yaml, or along side it, consult the following doccumentation:
+https://www.terraform.io/docs/providers/openstack/#configuration-reference
+
+
+## OpenStack Requirements
+
+### Recommended Minimums
+
+With the latest version of the installer, we recommend the following minimums in order to get a succesful deployment.
+
+   * OpenStack Quota
+     * Networking
+       * Floating IPs: 2
+       * Security Groups: 2
+       * Security Group Rules: 44
```

Comment:
My deployment had 30 master rules and 18 worker rules for a total of 48. It's possible that more will be added in the future. I'd set the minimum to say 60 and mention that ~100+ would be recommended for future-proofing.

Note that Kuryr will have much higher requirements on the networking resources (but that can be addressed later by the Kuryr folks).